test/src/page.spec.ts

```diff
@@ -111,6 +111,28 @@ describe('Page', function () {
         expect(message).not.toContain('Timeout');
       }
     });
+    it('closing last tab should not close the browser', async () => {
+      // This test asserts the behavior is consistent between protocols and browsers.
+      // WebDriver Classic requires the session to be closed after the last page is
```

Comment:
Not sure if we need the `WebDriver Classic` comment as Puppeteer does not use it.